source/jquery.flot.navigate.js

```diff
@@ -354,17 +355,23 @@ can set the default in the options.
 
         function onDblClick(e) {
             plot.activate();
-
-            var axes = plot.getTouchedAxis(e.clientX, e.clientY),
-                event;
-            if (axes[0]) {
-                event = new $.Event('re-center', { detail: {
-                    axisTouched: axes[0]
-                }});
-            } else {
-                event = new $.Event('re-center', {detail: e});
+            var o = plot.getOptions()
+
+            if (o.pan.interactive && o.pan.allowRecenter) {
```

Comment:
This outer if should completely go away, and we should have the following clause before the rest of the implementation:

`if (!o.recenter.interactive) { return; }`

Reply:
Done.